randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1455,21 +1450,40 @@ Extra - map_name: rm_a5b05
   > Tunnel to Spiky Spider Trial Reward Room
       Any of the following:
           All of the following:
-              Morph Ball and Mid-Air Morph (Advanced)
+              # Suitless Method
+              Morph Ball
+              Any of the following:
+                  # Get past crumble blocks
+                  Mid-Air Morph (Advanced)
+                  Power Grip and Movement (Intermediate)
               Any of the following:
+                  # To climb the room
                   All of the following:
                       Walljump (Intermediate)
-                      Power Grip or Mid-Air Morph (Expert)
+                      Power Grip or Mid-Air Morph (Advanced)
                   All of the following:
                       # IBJ
                       Infinite Bomb Jumping (Advanced) and Can Use Bombs
-          Gravity Suit and Can Use Spider Ball
+          All of the following:
+              # Gravity Suit Method
+              Gravity Suit
+              Any of the following:
+                  # Get past crumble blocks
+                  Can Use Spider Ball
+                  All of the following:
+                      Hi-Jump Boots and Spring Ball and Movement (Intermediate)
+                      Any of the following:
+                          # To climb the room
+                          Hijump Tunnel Climb
+                          Hi-Jump Boots and Mid-Air Morph (Intermediate)
 
 > Tunnel to Spiky Spider Trial Reward Room; Heals? False
   * Layers: default
   * Morph Ball Tunnel to Spiky Spider Trial Reward Room/Tunnel to Spiky Spider Trial
   > Dock to Serris Arena Access
-      Trivial
+      All of the following:
```

Comment:
I'm wondering whether a comment here as to why it's not tunnel climb would be useful

Reply:
I don't think it needs justification as then that precedent would probably need to be flowed throughout the DB of explaining why preset X was used instead of Y. also, I could put an option for `Gravity+Spider`.

Reply:
fair enough